cmd/ocsp-updater/main.go

```diff
@@ -207,7 +207,10 @@ func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Ti
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT cs.*
+		`SELECT
```

Comment:
You should also remove the JOIN in this query, to demonstrate the query as it will look after this optimization (but keep that to its own commit so you can split it off easily).

Similarly for the other query we wanted to optimize with these changes.